docs_src/V2TopLevelMigration.md

```diff
@@ -0,0 +1,165 @@
+# V2 Migration Guide
+
+!!! edgey "EdgeX 2.0"
+    Many backward breaking changes occurred in the EdgeX 2.0 (Ireland) release which may require some migration depending on your use case.
+
+This section describes how to migrate from EdgeX 1.x to EdgeX 2.0 at a high level and refers the reader to the appropriate detail documents. The areas to consider for migrating are:
+
+- [Custom Compose File](#custom-compose-file)
+- [Database](#database)
+- [Custom Configuration](#custom-configuration)
+- [Custom Device Service](#custom-device-service)
+- [Custom Device Profile](#custom-device-profile)
+- [Custom Pre-Defined Device](#custom-pre-defined-device)
+- [Custom Applications Service](#custom-applications-service)
+- [Security](#security)
+
+## Custom Compose File
+
+The compose files for V2 have many changes from their V1 counter parts. If you have customized a V1 compose file to add additional services or tweak overrides,  it is highly recommended that you start with the appropriate V2 compose file and re-add your customizations. It is very likely that the sections for your additional services will need to be migrated to have the proper environment overrides. Best approach is to use one of the V2 service sections that closest matches your service  as a template.
```

Comment:
I originally was going to comment on your usage of `V1` and `V2` throughout this document, but I decided not to... however after just re-reading your intro which uses `1.x `and `2.0` instead, I would suggest using this across the board (or noting that for purposes of brevity, you'll use V1/V2 everywhere else).

Reply:
suggest changing "or tweak overrides" to "and/or add or modify configuration overrides"